<launch>

<rosparam file="$(find agree_fsm_pkg)/config/exercise.yaml" />
<rosparam file="$(find agree_fsm_pkg)/config/general.yaml" />
<rosparam file="$(find agree_fsm_pkg)/config/physiological_param.yaml" />
<rosparam file="$(find agree_fsm_pkg)/config/evaluation.yaml" />
<rosparam file="$(find agree_fsm_pkg)/config/test.yaml" />

<node name="shared_memory_interface" pkg="agree_esmacat_pkg" type="agree_ros_dummy" output="screen" />
</launch>
